exercises/D-logical-operators/exercise2.js

```diff
@@ -5,8 +5,30 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(number) {
+  return (number < 0) ?  true : false;
+}
 
+function isBetween5and10(number) {
+  if (number >= 5 && number <=10){
+    return true;
+  }
+  else {
+    return false;
+  }
+}
+
+function isShortName(name) {
+  const nameLength = name.length;
+  if (nameLength <=6){
+    return true
+  }
+  return false;
+}
+
+function startsWithD(name) {
+  return name.charAt(0) === "D" || name.charAt === "d";
+}
```

Comment:
typo in the second part of the condition, charAt needs an index
the string's startsWith function can be used as well along with the toLowerCase/toUpperCase function